### ISSUE TITLE
Addins aren't loading in Excel WOPI

### ISSUE THREAD

**Issue by SirDavenport** [OfficeDev/office-js#4445]
# Provide required information needed to triage your issue
When one person opens an Excel document in Wopi, the addin thats supposed to load from host_install_addins loads fine. However, when a second person opens that same excel document in WOPI, the addin does not load. This does not happen with word, and only just recently started happening with excel.

## Your Environment
PC. Microsoft Edge
 
## Expected behavior
The Addins loaded through host_install_addins in WOPI should load for anyone that opens the document.

## Current behavior
The addin is only loading for the first person to open the document. Then when the second person opens the document, the addin doesn't load.

## Steps to reproduce
1. Have a wopi implementation that loads an addin through host_install_addins
2. Open an excel document in Wopi and watch the addin load.
3. As the same user or a different user, open the same Excel document and see that the addin does not load.

**Comment by shanshanzheng-dev** [comment]
Hi @SirDavenport, Thanks for reporting this issue. Could you please tell us more details about how to repro this issue? Such as which add-in has this issue? Thanks.

**Comment by SirDavenport** [comment]
Hi thanks for taking a look. The addin we are loading through WOPI is id WA200003648. And again, this is only happening in Excel in WOPI when a second user is opening the document. The addin loads for the first user.

**Comment by shanshanzheng-dev** [comment]
Hi @SirDavenport Thanks for this details. Sorry, I can't repro this issue. Are you get this addin from AppSource and share someone the doc? like: https://appsource.microsoft.com/en-us/product/office/WA200003648?tab=Overview. Is it a solid repro in your side? If you can identify a user condition or have a solid repro please feel free to comment here again that will help our investigation and get sooner to the issue.

**Comment by SirDavenport** [comment]
Hi @shanshanzheng-dev, you are using a WOPI implementation, right? The issue is not reproducable in the desktop. Only Wopi in Office Online. If you want, I can private message you and give you access to one of our environments, because I can reproduce every time I try there.

**Comment by SirDavenport** [comment]
Hi @shanshanzheng-dev, Hope you are having a good week. Any update on this?

**Comment by shanshanzheng-dev** [comment]
HI @SirDavenport Sorry for slow response. We're looking into this issue, will report back here if we have a suggestion for you. Thanks.

**Comment by shanshanzheng-dev** [comment]
Hi @SirDavenport Could you share us a test account? Thanks.

**Comment by SirDavenport** [comment]
Hi, @shanshanzheng-dev. Hope you had a good weekend. I'll need an email address of the person that wants the test account.

**Comment by shanshanzheng-dev** [comment]
Thanks @SirDavenport. My email address is szheng@microsoft.com

**Comment by SirDavenport** [comment]
Thanks @shanshanzheng-dev, you should have received a couple emails.

**Comment by shanshanzheng-dev** [comment]
Thanks @SirDavenport, I have received emails and we'll look into this issue. But when we click that button, it just takes me to Google homepage...
![image](https://github.com/OfficeDev/office-js/assets/125014206/a16f4400-1fff-4e08-a510-154ca8a49aec)

**Comment by phtai** [comment]
@SirDavenport, could you please also share a test account with me? I will investigate this issue. My email is my github user name + @microsoft.com.

**Comment by SirDavenport** [comment]
@phtai I just added you. You should have received a couple emails. @shanshanzheng-dev Sorry about that, just sent you another email.

**Comment by phtai** [comment]
Thanks @SirDavenport. I have successfully repro the issue with my test account yesterday. However, today when I try again, the add-in always launches, even after I refreshed the new tab many times. Does the bug still happen to you now?

**Comment by SirDavenport** [comment]
It's intermittent. Sometimes I have to close out and start coauthoring
again. Remember, when it's one person, it loads every time, as soon as a
second session loads the same document, that's when the issue sometimes
happens. So you might want to make sure your other session didn't log out

**Comment by phtai** [comment]
Yes, I understand that and tried again today, but I still can't repro. I kept multiple tabs of the doc opened and tried multiple times.

We recently have rolled out some bug fixes. While those bugs are not directly related to this scenario, it is possibly they are indirectly causing this issue, so I would like to clarify the situation here.

@SirDavenport, when is your latest repro of the issue? Is it still happening now?

**Comment by SirDavenport** [comment]
I am on vacation. As soon as I get back I'll try and reproduce and let you
know.

Thanks,
Derek

**Comment by SirDavenport** [comment]
Hi phtai,
Just got back and tested it, and the issue happened on the first try. I
logged in, opened the excel document RocketDocs
<https://qa.rocketdocs.com/app/office?contextId=1374&contextType=projectdocument>,
then I opened the tab again from either going to the Project page and
clicking it, or duplicating tab, and the second instance of the WOPI did
not load the addin.

Thanks,
Derek

**Comment by SirDavenport** [comment]
@phtai Hope you are having a good day. Did you see my email? Just wondering if there is any update on this issue.

**Comment by phtai** [comment]
@SirDavenport Yes, I saw your email, but unfortunately, I am still unable to repro the issue. Would you mind capturing a browser network trace where a new tab doesn't load the add-in for me to debug? You can send it to my email (my github user name @microsoft.com)

**Comment by phtai** [comment]
@SirDavenport I just had a repro for the issue. It is rare for me, but I successfully got one, so I don't need a network trace from you. From my initial investigation it is an issue coming from excel online server redirection logic. I will continue to find the root cause and keep you updated.

**Comment by SirDavenport** [comment]
Hi @phtai thanks for the update. I look forward to hearing about what you find out!

**Comment by SirDavenport** [comment]
Howdy @phtai , hope you are having a good week. Do you have any update on this issue?

**Comment by SirDavenport** [comment]
@shanshanzheng-dev @phtai Good morning. Just checking in to see if there is any update on this issue.

**Comment by SirDavenport** [comment]
Howdy @shanshanzheng-dev @phtai @m-hellesen ,
Just checking in to see if there has been any update on this issue.

Thanks

**Comment by SirDavenport** [comment]
@phtai Any update on this issue? One thing I just noticed is, when I am the only person in the document, the urls are https://excel.officeapps.live.com, but when there is someone else in there, they are https://pus8-excel.officeapps.live.com, and when they are the latter, the addin does not load.

**Comment by shanshanzheng-dev** [comment]
Hi @phtai could you help take a look?
@SirDavenport about the another other, seem it's a known issue, we're investigating it.

**Comment by jaiswalshubh1** [comment]
Hi @SirDavenport ,

As per the internal ticket: 8985332, The issue has been fixed, Can you please verify from your side ?

CC: @kustonaut 

Thanks!
Shubham Jaiswal
